genref/.goreleaser.yml

```diff
@@ -0,0 +1,25 @@
+builds:
+  - env:
+      - CGO_ENABLED=0
+      # travis ci currently sets GOPATH even with go1.11.
+      # force-setting GO111MODULE=on to use vgo
+      - GO111MODULE=on
+    goos:
+      - linux
+      - darwin
+    goarch:
```

Comment:
I did not know `travis` is in available?

Reply:
copied from previous repo, should be removed from this PR.